docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -237,3 +239,7 @@ The `yb_get_current_transaction_priority` function shows the transaction priorit
     Highest priority transaction
     (1 row)
     ```
+
+## Learn more
```

Comment:
"Learn more", "Next steps", etc, should never be a header. Instead, there should be a "For more info" paragraph, typically under the top section of the page.

For more information, see the following:

- link 1
- link 2
- link 3